environments/prod/apply-divorce-service-gov-uk.yml

```diff
@@ -43,6 +43,19 @@ cname:
   - name: "cdnverify.www.decree-nisi"
     ttl: 3600
     record: "cdnverify.hmcts-decree-nisi-shutter-prod.azureedge.net"
+  - name: "manage-divorce-application"
+    ttl: 300
+    record: "www.manage-divorce-application.service.gov.uk"
+  - name: "www.manage-divorce-application"
```

Comment:
the url will be:

www.manage-divorce-application.apply-for-divorce.service.gov.uk

that seems a bit excessively long is this what you intended?

Reply:
That's what the service team have agreed